src/app/utils/functions/is-alpha-numeric.ts

```diff
@@ -6,6 +6,6 @@ export const isAlphaNumericOnly = (event: any, includeDot?: boolean): boolean =>
     (charCode > 96 && charCode < 123) || // a-z
     (charCode > 64 && charCode < 91) || // A-Z
     (charCode > 47 && charCode < 58) || // 0-9
-    (includeDot && charCode === 46) // .
+    (!!includeDot && charCode === 46) // .
```

Comment:
why? It is already a boolean value.

Reply:
event "keypress" need only true or false if we return undefined it is not work correct

Reply:
Oh, ok. That's why we need tests for such cases...